cmd/kind/delete/cluster/deletecluster.go

```diff
@@ -25,7 +25,8 @@ import (
 )
 
 type flags struct {
-	Name string
+	Name  string
+	Debug bool
```

Comment:
/hold
I don't think the flag should literally be `debug`, and I'm having second thoughts about even having this flag, it's sort of a weird mode to enable to begin with. cc @munnerz

Reply:
> /hold
> I don't think the flag should literally be `debug`, and I'm having second thoughts about even having this flag, it's sort of a weird mode to enable to begin with. cc @munnerz

Sure, we can have other names or modify this behavior. 
Or modify this `retention mode` to a global scope? Bind this behavior to log level? 

I want to know your thoughts. Thanks.

Reply:
ok, it can also clearly indicate the meaning.

Reply:
I will start to modify it later.